docs/guides/model_surgery.md

```diff
@@ -0,0 +1,140 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+Model Surgery
+==============================
+
+Usually, Flax modules and optimizers track and update the params for you. But there may be some time when you want to do some model surgery and tweak the param tensors yourself. This guide shows you how to do the trick.
+
+
+## Setup
+
+```python
+!pip install --upgrade -q pip jax jaxlib flax
```

Comment:
This type of cells should be tagged with `skip-execution` to avoid installing dependencies on CI (could alter tests). See [myst-nb cell-tags](https://myst-nb.readthedocs.io/en/latest/configuration.html#cell-tags).

Reply:
Done. This is such a useful feature - we should make sure to have it on every doc.